js/models/messages.js

```diff
@@ -1031,65 +1031,52 @@
 
       const quoteWithData = await loadQuoteData(this.get('quote'));
       const previewWithData = await loadPreviewData(this.get('preview'));
-
+      const chatMessage = new libsession.Messages.Outgoing.ChatMessage({
+        body,
+        timestamp: this.get('sent_at'),
+        attachments,
+        quote: quoteWithData,
+        preview: previewWithData,
+        expireTimer: this.get('expireTimer'),
+      });
       // Special-case the self-send case - we send only a sync message
       if (recipients.length === 1 && recipients[0] === this.OUR_NUMBER) {
```

Comment:
i think this case can also be simplified to
```
if (recipients.length === 1 && recipients[0] === this.OUR_NUMBER) {
    await this.markMessageSyncOnly();
}
```
and then let the case below send it using the queue which automatically handles sync messages.

Reply:
updated